cocos/core/renderer/scene/directional-light.ts

```diff
@@ -102,9 +102,10 @@ export class DirectionalLight extends Light {
         this._type = LightType.DIRECTIONAL;
     }
 
-    public initialize() {
+    public initialize () {
         super.initialize();
-        LightPool.set(this._handle, LightView.ILLUMINANCE, Ambient.SUN_ILLUM);
+        this.illuminance = Ambient.SUN_ILLUM;
+        this.direction = this._dir;
```

Comment:
This is not a good idea, if one day someone decide to do parameter check in direction setter, then the initialize process will be broken unintentionally.

Please explicitly set the LightView direction in pool